action.yml

```
@@ -225,6 +273,29 @@ runs:
 
         TERRAFORM_RESULT=$?
 
+        if [[ "${{ inputs.pr-comment }}" == "true" ]]; then
+          tfcmt \
+          --config ${{ github.action_path }}/config/summary.yaml \
```

```suggestion
          --config ${GITHUB_ACTION_PATH}/config/summary.yaml \
```

because https://github.com/cloudposse/github-action-atmos-terraform-plan/pull/91